spec/wasabi/document/escaped_endpoint_spec.rb

```diff
@@ -0,0 +1,15 @@
+
+require "spec_helper"
+
+describe Wasabi::Document do
+  context "with: escaped_endpoint.wsdl" do
+
+    subject { Wasabi::Document.new fixture(:escaped_endpoint).read }
+
+    describe '#endpoint' do
+      subject { super().endpoint }
+      it { is_expected.to eq URI("http://localhost/soap%20service") }
```

Comment:
```suggestion
      it "supports spaces in URIs" { expect(document.endpoint).to eq URI("http://localhost/soap%20service") }
```

Perhaps something along these lines would make it "more concrete".

What do you think?